### ISSUE TITLE
Custom tags for planner

### ISSUE THREAD

**Issue by thatoddmailbox** [PlanHubMe/PlanHub#26]
You should be able to define custom tags.

Also use this issue to suggest more tag ideas.

**Comment by medensqui** [comment]
_Ahem_ Idea credit.

Also should I show people our rough beta?

I showed Malcolm and he said, "So basically a copy of Planbook" 

RESULT WE NEED MY DAY BY DAY 1

**Comment by medensqui** [comment]
@thatoddmailbox

**Comment by thatoddmailbox** [comment]
@medensqui 

> I showed Malcolm and he said, "So basically a copy of Planbook"

DON'T SHOW IT TO PEOPLE OR THEY THINK THAT AND THEN EVERYONE WILL BE LIKE "oh no it's that copy" AND NO ONE WILL USE IT AAAAGGGH

But yes, we need #1 and maybe #9 to set it apart.

**Comment by medensqui** [comment]
@thatoddmailbox ok

Yes pls

Some features that can set Labby apart from youcanbook (I'm assuming Labby will become a part of PlanHub)
- Gmail notifications
- More customization
- Maybe use Bootstrap?
- Get rid of captcha

**Comment by medensqui** [comment]
@thatoddmailbox And also how the flip do you quote stuff in chat

Also, I explained My Day to Malcolm, no reaction.

**Comment by medensqui** [comment]
@thatoddmailbox So idk

**Comment by thatoddmailbox** [comment]
@medensqui 

I think we should make Labby part of planhub and have a happy API for CoursesPlus and other things.

Quote things by putting an arrow ">" in front of the line.

> Also, I explained My Day to Malcolm, no reaction.

You would want a thing to show than lots of words...

**Comment by medensqui** [comment]
> You would want a thing to show than lots of words...

Yes LETS START WORKING ON IT SO THAT WE CAN MAKE PLANHUB PUBLIC SUPER SOON

**Comment by medensqui** [comment]
OOPS @thatoddmailbox WHAT DID I DO

**Comment by thatoddmailbox** [comment]
@medensqui YOU HAVE TO LEAVE A BLANK LINE BETWEEN THE QUOTE AND YOUR COMMENT BECAUSE MARKDOWN

**Comment by medensqui** [comment]
@thatoddmailbox OH SO DO YOU AGREE START WORKING ON IT
